Open Judge System/Web/OJS.Web/Areas/Contests/Views/Results/Simple.cshtml

```diff
@@ -1,28 +1,22 @@
 ﻿@model OJS.Web.Areas.Contests.ViewModels.Results.ContestResultsViewModel
 
 @using OJS.Common.Models
+@using OJS.Web.Areas.Administration.Controllers
 @using OJS.Web.Areas.Contests.Controllers
-@using OJS.Web.ViewModels.Shared
+
 @using Resource = Resources.Areas.Contests.Views.ResultsSimple
 
 @{
-    ViewBag.Title = string.Format(Resource.Title, Model.Name, ViewBag.IsOfficial ? string.Empty : " (Practice)");
-    ViewBag.Subtitle = ViewBag.IsOfficial ? string.Empty : Resource.CachedResults;
-    string isOfficial = ViewBag.IsOfficial ? CompeteController.CompeteActionName : CompeteController.PracticeActionName;
-    var resultsOnPage = ViewBag.IsOfficial
-        ? ResultsController.OfficialResultsPageSize
-        : ResultsController.NotOfficialResultsPageSize;
-
-    var results = Model.Results.ToList();
+    ViewBag.Title = string.Format(Resource.Title, Model.Name, Model.IsCompete ? string.Empty : " (Practice)");
+    ViewBag.Subtitle = Model.IsCompete ? string.Empty : Resource.CachedResults;
 }
 
 @section Styles {
     <link href="/Content/Contests/results-page.css" rel="stylesheet" />
```

Comment:
Add the styles from a bundle instead of manually.

Reply:
Done in https://github.com/SoftUni-Internal/OpenJudgeSystem/pull/223